src/app/services/sms/sms.service.ts

```diff
@@ -461,3 +465,32 @@ export const sendBouncedSubmissionSms = (
     SmsType.BouncedSubmission,
   )
 }
+
+/**
+ * Retrieves the free sms count for a particular user
+ * @param userId The id of the user to retrieve the sms counts for
+ * @returns ok(count) when retrieval is successful
+ * @returns err(error) when retrieval fails due to a database error
+ */
+export const retrieveFreeSmsCounts = (
+  userId: string,
+): ResultAsync<number, PossibleDatabaseError> => {
+  return ResultAsync.fromPromise(
+    SmsCount.countDocuments({
+      'formAdmin.userId': userId,
+      smsType: SmsType.Verification,
+      isOnboardedAccount: false,
+    }).exec(),
```

Comment:
should this be wrapped as a SmsCount's model instance method instead?

Reply:
TIL: statics define a method on the model rather than on the schema